pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -4150,14 +4237,15 @@ func TestEnvCustomImageTag(t *testing.T) {
 	checkImageNames(cr, imageName, imageURL, t)
 
 	// test setting image with env vars, DM product, and different image url
+
 	cr.Spec.Environment = api.RhdmAuthoring
 	image = "testing"
 	imageTag = cr.Status.Applied.Version
 	imageName = image + ":" + imageTag
 	imageURL = "image-registry.openshift-image-registry.svc:5000/openshift/" + imageName
 	envConstants = constants.EnvironmentConstants[cr.Spec.Environment]
 	os.Setenv(envConstants.App.ImageVar+cr.Status.Applied.Version, imageURL)
-	os.Setenv(constants.DmKieImageVar+cr.Status.Applied.Version, imageURL)
+	os.Setenv(constants.PamKieImageVar+cr.Status.Applied.Version, imageURL)
 	os.Setenv(constants.PamSmartRouterVar+cr.Status.Applied.Version, imageURL)
 	checkImageNames(cr, imageName, imageURL, t)
```

Comment:
@spolti here fail because https://github.com/kiegroup/kie-cloud-operator/blob/3b776ffc1b45e9efc708903f1947acc3be08d4de/pkg/controller/kieapp/defaults/defaults.go#L415 isn't found and the template.ImageURL isn't overrided